errors/navigation-context-missing.md

```diff
@@ -0,0 +1,34 @@
+# Missing Navigation Context
+
+#### Why This Error Occurred
+
+This can happen when you have components that are shared between `pages/` and `app/` or were trying to use the `next/navigation` hooks outside the `app/` directory. Situations where you also render components outside a Next.js application (like in unit tests) can also trigger this error.
+
+The following hooks can be affected by missing contexts:
+
+- `useSearchParams()`
+- `usePathname()`
+
+When pages are automatically statically optimized (See [Automatic Static Optimization](https://nextjs.org/docs/advanced-features/automatic-static-optimization)) or are rendered as a fallback page (See [Fallback Pages](https://nextjs.org/docs/api-reference/data-fetching/get-static-paths#fallback-pages)) the `useSearchParams()` and `usePathname()` hooks will return `null` until hydration occurs.
```

Comment:
Automatically Statically Optimized? (it's branding so properly capitalized?)

Reply:
This wording is already used in other documentation available, see https://nextjs.org/docs/api-reference/next/router for an example.